src/materials/MeshLambertMaterial.js

```diff
@@ -46,28 +46,30 @@ function MeshLambertMaterial( parameters ) {
 
 	this.type = 'MeshLambertMaterial';
 
-	this.color = new Color( 0xffffff ); // diffuse
+	this.isExperimentalMaterial = true;
 
-	this.map = null;
+	this.addParameter( 'color', new Color( 0xffffff ), 'diffuse' ); // diffuse
```

Comment:
Maybe a less intrusive approach could be something like... `this.defineParameters( this, [ 'color', 'lightMap', 'lightMapIntensity', ... ] );